src/main/webapp/index.jsp

```diff
@@ -7,133 +7,122 @@
 <t:staticPage jsIncludes="${jsIncludes}" currentPage="index">
```

Comment:
Since we are using main tag in the other pages, we should do the same for this, what do you think?